### PR TITLE
Minor translation changes in homepage

### PR DESCRIPTION
**Pull request by icarlossz**
Hey guys! 

I was checking the Spanish translation of the homepage and I saw some little things that weren't translated in the same way as they were in the original version so I wanted to correct it.

Suggestions/comments are accepted.

### CODE REVIEW

File: content/home/examples/a-simple-component.md [hunk 1]
```diff
@@ -4,6 +4,6 @@ order: 0
 domid: hello-example
 ---
 
-Los componentes de React implementan un método llamado`render()` que recibe datos de entrada y los muestra. Este ejemplo utiliza una sintaxis similar a XML llamada JSX. Puedes acceder a los datos de entrada que se pasan al componente mediante `render()` a través de `this.props`.
+Los componentes de React implementan un método llamado `render()` que recibe datos de entrada y retorna que mostrar. Este ejemplo utiliza una sintaxis similar a XML llamada JSX. Puedes acceder a los datos de entrada que se pasan al componente mediante `render()` a través de `this.props`.
```
Comment (alejandronanez) [inline content/home/examples/a-simple-component.md]:
```suggestion
Los componentes de React implementan un método llamado `render()` que recibe datos de entrada y retorna qué mostrar. Este ejemplo utiliza una sintaxis similar a XML llamada JSX. Puedes acceder a los datos de entrada que se pasan al componente mediante `render()` a través de `this.props`.
```

File: src/pages/index.js [hunk 2]
```diff
@@ -154,7 +154,7 @@ class Home extends Component {
                     </CtaItem>
                     <CtaItem>
                       <ButtonLink to="/tutorial/tutorial.html" type="secondary">
-                        Haz el tutorial
+                        Toma el tutorial
```
Comment (alejandronanez) [inline src/pages/index.js]: Hey @icarlossz, these two are pretty much the same, let's leave them as they are for now.

```suggestion
                        Haz el tutorial
```

File: src/pages/index.js [hunk 3]
```diff
@@ -291,7 +291,7 @@ class Home extends Component {
                 </CtaItem>
                 <CtaItem>
                   <ButtonLink to="/tutorial/tutorial.html" type="secondary">
-                    Haz el tutorial
+                    Toma el tutorial
```
Comment (alejandronanez) [inline src/pages/index.js]: Same here, you mind rolling this back?
```suggestion
                    Haz el tutorial
```

File: content/home/examples/a-stateful-component.md [hunk 4]
```diff
@@ -4,4 +4,4 @@ order: 1
 domid: timer-example
 ---
 
-Además de obtener datos de entrada (a los que accedes a través de `this.props`),un componente puede tener datos en su estado internamente (a los que accedes a través de `this.state`). Cuando el estado del componente cambie de datos, el resultado se actualiza volviendo a invocar a `render()`.
+Además de obtener datos de entrada (a los que accedes a través de `this.props`), un componente puede tener datos en su estado interno (a los que accedes a través de `this.state`). Cuando los datos del estado de un componente cambian, el cambio va a ser actualizado reinvocando `render()`.
```
Comment (alejandronanez) [inline content/home/examples/a-stateful-component.md]: ```Cuando los datos del estado de un componente cambian, el cambio va a ser actualizado reinvocando `render()````

Sounds pretty ambiguous, what if we add something between the lines of 

```
Cuando los datos del estado de un componente cambian, se vuelve a invocar `render` con los nuevos valores en `this.state`
```

I know this is not an exact word by word translation, but makes more sense than what we have now.

Comment (icarlossz) [reply content/home/examples/a-stateful-component.md]: It sounds better.

File: src/pages/index.js (hunk 3 above)
Comment (icarlossz) [reply src/pages/index.js]: Done

File: src/pages/index.js (hunk 2 above)
Comment (icarlossz) [reply src/pages/index.js]: Done

File: content/home/examples/a-simple-component.md (hunk 1 above)
Comment (icarlossz) [reply content/home/examples/a-simple-component.md]: Fixed

**Comment by icarlossz** [comment]
Hey Alejandro, thanks for the review! I made your suggestions :)

**Pull request merged**